Hestia/app/src/main/java/hestia/backend/RemoveDeviceTask.java

```diff
@@ -0,0 +1,82 @@
+package hestia.backend;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/**
+ * This class will send a DELETE request to the Server in order to delete
+ * the Device "device".
+ */
+
+public class RemoveDeviceTask extends AsyncTask<Void, Void, Void> {
```

Comment:
I don't agree with you here. there is no point of returning a value. you may argue that it would be neater this way, but if you return a value, you would then want to store it somewhere in CIC. It wouldn't be nice that way. You will be using this value only for Logging, but Logging should be done inside the RemoveDeviceTask class.

Also, think of it like this. you are sending a DELETE request. you shouldn't be returning anything, right?